### PR TITLE
Issue #13321: Kill mutation for SummaryJavaDocCheck-5

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #13321: Kill mutation for SummaryJavaDocCheck-5

-----

# Check :- 
https://checkstyle.org/checks/javadoc/summaryjavadoc.html#SummaryJavadoc

-----

# Mutation 
https://github.com/checkstyle/checkstyle/blob/ce6a1113b574960e8b7e311af4c279aad24ed71b/config/pitest-suppressions/pitest-javadoc-suppressions.xml#L903-L910

-----

# Explaination
Loop run 1 time more

-----

# Regression :- 

Report-1 :- https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/7f8d3df_2023103809/reports/diff/index.html

Report-2 :- https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/7cb1b47_2023160922/reports/diff/index.html

-----

Diff Regression config: https://gist.githubusercontent.com/Kevin222004/61eaa69fe98585b6ee1882ca082362bc/raw/316ca02ce9fd290119d9bfc560f2b11e84a1b685/Summary.xml
Diff Regression projects: https://gist.githubusercontent.com/Kevin222004/21e3934e85f802e2fbd48af06d122364/raw/604256badd733d8568064f371d55657c04b00dfd/test-projects-2.properties
Report label: change-config-Regression-1

### CODE REVIEW

**Comment by Kevin222004** [comment]
Github, generate report

**Comment by Kevin222004** [comment]
Github, generate report

**Comment by Kevin222004** [comment]
Github, generate report

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java [hunk 1]
```diff
@@ -368,7 +368,7 @@ private static Optional<DetailNode> getInlineTagNode(DetailNode javadoc) {
      */
     private static boolean isDefinedFirst(DetailNode inlineSummaryTag) {
         boolean isDefinedFirst = true;
-        DetailNode previousSibling = JavadocUtil.getPreviousSibling(inlineSummaryTag);
+        DetailNode previousSibling = inlineSummaryTag;
         while (previousSibling != null && isDefinedFirst) {
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java:372]: Please rename to ` currentAst`

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Review by Vyom-Yadav — APPROVED** [review]
LGTM!

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**